Makefile

```
@@ -40,7 +40,7 @@ ifeq ($(OSTYPE), darwin)
 	GCOV = xcrun llvm-cov gcov
 endif
 
-BUILDTYPE_LOWER := $(shell echo $(BUILDTYPE) | tr '[A-Z]' '[a-z]')
+BUILDTYPE_LOWER := $(shell printf $(BUILDTYPE) | tr '[A-Z]' '[a-z]')
```

What do you think of using this syntax instead?
```suggestion
BUILDTYPE_LOWER := $(shell tr '[A-Z]' '[a-z]' <<< $(BUILDTYPE))
```

---

It runs into this error when I run it:
```console
/bin/sh: 1: Syntax error: redirection unexpected
```
How about this:
```suggestion
BUILDTYPE_LOWER := $(BUILDTYPE) | tr '[A-Z]' '[a-z]'
```
No `printf` needed here I think.